modules/howtos/pages/managing-connections.adoc

```diff
@@ -223,7 +246,9 @@ It is important to make sure you are transferring the certificate in an encrypte
 If you are running on `localhost` and just want to enable TLS for a development machine, just copying and pasting it suffices
 -- _so long as you use `127.0.0.1` rather than `localhost` in the connection string_.
 This is because the certificate will not match the name _localhost_.
-Setting `TLSSkipVerify` is a workaround if you need to use ` couchbases://localhost`.
+
+// TODO: Not entirely sure this is correct, needs to be verified.
+//Setting `TLSSkipVerify` is a workaround if you need to use `couchbases://localhost`.
```

Comment:
Good catch -- that is not correct.  It should mention [TLSVerifyMode](https://github.com/couchbase/couchbase-python-client/blob/6d302a313cea3ed67d162ffaf7a448677cdd1d8c/couchbase/logic/options.py#L203-L206) or adding ssl=no_verify to the connection string (although I think we want to deprecate the connection string option).  We can address that w/ another ticket 👍